src/lib/Components/Bidding/Screens/ConfirmBid.tsx

```diff
@@ -383,6 +401,9 @@ export const ConfirmBidScreen = createFragmentContainer(ConfirmBid, {
   me: graphql`
     fragment ConfirmBid_me on Me {
       has_qualified_credit_cards
+      bidders(sale_id: $saleID) {
+        qualified_for_bidding
```

Comment:
We could just retrieve the `id` here instead of `qualified_for_bidding`-- it's a followup whether or not we care about your registration status once you get to this page (since theoretically you shouldn't even be able to _enter_ the bid flow if your registration is pending).